### PR TITLE
Math magicians events

### PR DESCRIPTION
**Pull request by jash-debug**
- Add logic files
- Add events to the buttons and display screen
- add CSS styling for the screen
- build dependency files
- update descriptive readme

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,35 +1,51 @@
...
+          <button type="button" id="x" value="x" className="special" onClick={(e) => this.handleBtn(e.target.id)}>&times;</button>
+          <button type="button" id="4" value="4" onClick={(e) => this.handleBtn(e.target.id)}>4</button>
+          <button type="button" id="5" value="5" onClick={(e) => this.handleBtn(e.target.id)}>5</button>
+          <button type="button" id="6" value="6" onClick={(e) => this.handleBtn(e.target.id)}>6</button>
+          <button type="button" id="-" value="-" className="special" onClick={(e) => this.handleBtn(e.target.id)}>-</button>
+          <button type="button" id="1" value="1" onClick={(e) => this.handleBtn(e.target.id)}>1</button>
+          <button type="button" id="2" value="2" onClick={(e) => this.handleBtn(e.target.id)}>2</button>
+          <button type="button" id="3" value="3" onClick={(e) => this.handleBtn(e.target.id)}>3</button>
+          <button type="button" id="+" value="+" className="special" onClick={(e) => this.handleBtn(e.target.id)}>+</button>
+          <button type="button" id="3" value="3" className="zero" onClick={(e) => this.handleBtn(e.target.id)}>0</button>
```
Comment (Ismailco) [inline src/components/Calculator.js]: - [x] Kindly make sure to fix the `0` button so it's value is right, for now the value is `3`

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @jash-debug, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (jash-debug) [reply src/components/Calculator.js]: Resolved

**Review by sumairq — APPROVED** [review]
### Hi @jash-debug  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**